### PR TITLE
Adding Demo for Cloud Events Orchestration

### PR DESCRIPTION
**Pull request by salaboy**
Link providing a demo for Cloud Events orchestration. Follow up example will come with Knative.

### CODE REVIEW

**Comment by salaboy** [comment]
@duglin Hi Doug, Mauricio here.. I'm having a DCO issues in this PR, do you know how can I fix that?

File: community/demos.md [hunk 1]
```diff
@@ -2,6 +2,9 @@ If you have a demo of CloudEvents in action, please add a link here. If there
 isn't an associated blog or github repo, feel free to add descriptive text as a
 markdown file in `community/demos/`.
 
+- [Orchestrating Cloud Events](https://salaboy.com/2020/05/18/orchestrating-cloud-events-with-zeebe/)
+  - Blog post describing how Cloud Events can be orchestrated using an orchestration engine. Example included that you can run in your own environment with links to GitHub repositories needed.
```
Comment (duglin) [inline community/demos.md:6]: minor thing, but can you wrap this at 80 columns to align with the rest of the doc?

File: community/demos.md [hunk 2]
```diff
@@ -2,6 +2,9 @@ If you have a demo of CloudEvents in action, please add a link here. If there
 isn't an associated blog or github repo, feel free to add descriptive text as a
 markdown file in `community/demos/`.
 
+- [Orchestrating Cloud Events](https://salaboy.com/2020/05/18/orchestrating-cloud-events-with-zeebe/)
+  - Blog post describing how Cloud Events can be orchestrated using an orchestration engine. Example included that you can run in your own environment with links to GitHub repositories needed. 
+    - [Demo Code and instructions]: https://github.com/salaboy/zeebe-cloud-events-examples
```
Comment (duglin) [inline community/demos.md:7]: This line is funky, it shows up with just the bullet but no text:
image

I think you're missing the ()'s around the link

File: community/demos.md (hunk 1 above)
Comment (salaboy) [reply community/demos.md]: @duglin thanks a lot for the comment. Is there any way to make sure that it just breaks at 80 chars? I've tried several options but line-breaks will not happen inside bullets.

File: community/demos.md (hunk 2 above)
Comment (salaboy) [reply community/demos.md]: Fixed, thanks a lot!

File: community/demos.md [hunk 3]
```diff
@@ -2,6 +2,12 @@ If you have a demo of CloudEvents in action, please add a link here. If there
 isn't an associated blog or github repo, feel free to add descriptive text as a
 markdown file in `community/demos/`.
 
+- [Orchestrating Cloud Events](https://salaboy.com/2020/05/18/orchestrating-cloud-events-with-zeebe/)
+
+  - Description: Blog post describing how Cloud Events can be orchestrated using an orchestration engine. Example included that you can run in your own environment with links to GitHub repositories needed.
```
Comment (duglin) [inline community/demos.md:7]: can you wrap the lines at 80 chars?

File: community/demos.md [hunk 4]
```diff
@@ -2,6 +2,12 @@ If you have a demo of CloudEvents in action, please add a link here. If there
...
+  - Instructions: You can find the [Demo Code and instructions here](https://github.com/salaboy/zeebe-cloud-events-examples).
+  - Tags: Cloud Events, Java, Orchestration, Workflow, KNative, Example
```
Comment (duglin) [inline community/demos.md:9]: Why this line? None of the other ones have this

File: community/demos.md (hunk 1 above)
Comment (duglin) [reply community/demos.md]: just add a new line and then indent the next line, see https://github.com/cloudevents/spec/pull/646/files#diff-4c22f53fb332a4e6f2c6a9ecd53283aaL17

File: community/demos.md (hunk 4 above)
Comment (salaboy) [reply community/demos.md]: line removed to avoid a mismatch with other entries

**Pull request merged**